inst/extdata/OSD/A/ARGENT.json

```diff
@@ -97,8 +97,8 @@
         "dry_value": "NA",
         "dry_chroma": "NA",
         "moist_hue": "10YR",
-        "moist_value": 6,
-        "moist_chroma": 8,
+        "moist_value": 5,
+        "moist_chroma": "NA",
```

Comment:
Previously the matrix color returned here was the F3M colors -- the first valid color in the horizon narrative (10YR 6/8). Now pattern brings in the _correct_ matrix color `10YR 5/l`, but conversion of `"l"` to numeric results in NA chroma rather than `1`. With a quick post-processing function before output this would be 1 not NA